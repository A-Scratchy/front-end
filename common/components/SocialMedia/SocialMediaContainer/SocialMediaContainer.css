.SocialMediaContainer {
  display: flex;
  justify-content: space-around;
  align-items: center;
}

@media screen and (min-width: 320px) and (max-width: 768px) {
  .SocialMediaContainer {
    flex-wrap: wrap;
  }
}
