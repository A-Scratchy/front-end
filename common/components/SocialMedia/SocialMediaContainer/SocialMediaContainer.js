import React from 'react';
import PropTypes from 'prop-types';
import styles from './SocialMediaContainer.css';

const SocialMediaContainer = ({ children }) => (
  <div className={styles.SocialMediaContainer}>
    {children}
  </div>
);

SocialMediaContainer.propTypes = {
  children: PropTypes.arrayOf(PropTypes.element).isRequired
};

export default SocialMediaContainer;
