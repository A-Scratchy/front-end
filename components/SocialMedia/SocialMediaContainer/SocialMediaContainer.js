import { arrayOf, element } from 'prop-types';
import React from 'react';
import styles from './SocialMediaContainer.css';

SocialMediaContainer.propTypes = { children: arrayOf(element).isRequired };

function SocialMediaContainer({ children }) {
  return <div className={styles.SocialMediaContainer}>{children}</div>;
}

export default SocialMediaContainer;
