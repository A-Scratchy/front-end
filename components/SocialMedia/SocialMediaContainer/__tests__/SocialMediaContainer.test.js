import React from 'react';
import createSnapshotTest from 'test-utils/createSnapshotTest';

import SocialMediaContainer from '../SocialMediaContainer';

describe('SocialMediaContainer', () => {
  it('it should render properly with required (all) props', () => {
    createSnapshotTest(
      <SocialMediaContainer>
        <div>Testing 1</div>
        <div>Testing 2</div>
      </SocialMediaContainer>,
    );
  });
});
