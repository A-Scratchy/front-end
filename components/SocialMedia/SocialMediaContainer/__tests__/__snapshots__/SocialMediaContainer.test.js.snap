// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialMediaContainer should render with required props 1`] = `
<div
  className="SocialMediaContainer"
>
  <div>
    Testing 1
  </div>
  <div>
    Testing 2
  </div>
</div>
`;
